vitest.config.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path'
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    name: 'valtio',
+    setupFiles: './tests/setup.ts',
+    coverage: {
+      reporter: ['text', 'json', 'html', 'text-summary'],
+      reportsDirectory: './coverage/',
+    },
+    environment: 'jsdom',
+    dir: 'tests',
+    alias: [
+      {
+        find: /^valtio$/,
+        replacement: './src/index.ts'
+      },
+      {
+        find: /^valtio\/(.*)$/,
+        replacement: path.resolve(__dirname, './src/$1.ts')
```

Comment:
Can I just give a try with reverting this?
```suggestion
        replacement: './src/$1.ts'
```

If that doesn't work let's revert.